documentation/testing-docs/repo-tasks-module.md

```diff
@@ -0,0 +1,22 @@
+# Repo-Tasks Module
+
+## Usage:
+
+1. Double-click the shortcut located at `.\tools\PS-VSPrompt.lnk`, which will start VS Developer Prompt in PowerShell
+2. Run the command `Import-Module .\Repo-Tasks.psd1` to import the `Repo-Tasks` module
+	- During import, we allow users to load additional functions that they might want to use it in their session
+	- If you have a `userPreference.ps1` file under the `%userprofile%/psFiles` directory, the `Repo-Tasks` module will try to load it by dot sourcing it
+	- The module will also honor the environment variable `$env:psuserpreferences` and load `.ps1` files from the location that is pointed by `$env:psuserpreferences`
+	- As long as you have exported all functions that you need from your `.ps1` file using `Export-ModuleMember -Function {{functionName}}`. We deliberately do this to avoid polluting the list of commands available (when you use `Get-Command`)
+3. Currently, the `Repo-Tasks` module supports the following tasks:
+	- `Set-TestEnvironment`
```

Comment:
Set-TestEnvironment and New-TestCredential

Reply:
@maddieclayton fixed